ci_images/rhel-9/ci-openshift-golang-builder-latest/Dockerfile

```diff
@@ -1,6 +1,5 @@
-FROM replaced-by-buildconfig
```

Comment:
Create new file instead of renaming. Don't break existing transform mechanism until these images are building / mirroring / tested.